tests/e2e/workloads/app/amq/test_rgw_kafka_notifications.py

```diff
@@ -33,6 +34,7 @@
 @bugzilla("1958818")
 @on_prem_platform_required
 @skipif_external_mode
+@skipif_ocs_version(">=4.13")
```

Comment:
Instead, let's add the Bugzilla marker. This way, the test will be automatically picked for execution once the BZ is fixed.

Reply:
Done